enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/util/JWTUtils.java

```diff
@@ -161,11 +161,11 @@ public static PrivateKey getPrivateKey(String filePath) throws EnforcerException
             String strKeyPEM;
             Path keyPath = Paths.get(filePath);
             String key = Files.readString(keyPath, Charset.defaultCharset());
-            String lineSeparator = System.lineSeparator();
-            
+
             strKeyPEM = key
                     .replace(Constants.BEGINING_OF_PRIVATE_KEY, "")
-                    .replaceAll(lineSeparator, "")
+                    .replaceAll("\r\n", "") // for certs created in a Windows platform
```

Comment:
Then shall we just replace all "\n" and "\r".
Since this is a base64 encoded content and we know for sure there will not be any harm just replacing all "\n" and "\r".
@AmaliMatharaarachchi, @VirajSalaka wdyt?

```
.replaceAll("\n", "").replaceAll("\r", "")
```

Reply:
+1

Reply:
Resolved with https://github.com/wso2/product-microgateway/pull/3090/commits/75a0f5116a63b405ed7fc8dc31e81eff4b42e987